integration-authorization/src/main/java/org/symphonyoss/integration/authorization/oauth/OAuthRsaSignerFactory.java

```diff
@@ -1,95 +0,0 @@
-package org.symphonyoss.integration.authorization.oauth;
-
-import com.google.api.client.auth.oauth.OAuthRsaSigner;
-import com.google.api.client.repackaged.org.apache.commons.codec.binary.Base64;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
-import org.symphonyoss.integration.authorization.oauth.v1.OAuth1Exception;
-import org.symphonyoss.integration.logging.LogMessageSource;
-
-import java.security.KeyFactory;
-import java.security.NoSuchAlgorithmException;
-import java.security.PrivateKey;
-import java.security.PublicKey;
-import java.security.spec.InvalidKeySpecException;
-import java.security.spec.PKCS8EncodedKeySpec;
-import java.security.spec.X509EncodedKeySpec;
-
-/**
- * Deals with public and private RSA keys.
- *
- * Created by campidelli on 25-jul-2017.
- */
-@Component
-public class OAuthRsaSignerFactory {
-
-  private static final String RSA_NOT_FOUND = "integration.authorization.rsa.notfound";
-
-  private static final String RSA_NOT_FOUND_SOLUTION = RSA_NOT_FOUND + ".solution";
-
-  private static final String INVALID_PRIVATE_KEY = "integration.authorization.invalid.privatekey";
-
-  private static final String INVALID_PRIVATE_KEY_SOLUTION = INVALID_PRIVATE_KEY + ".solution";
-
-  private static final String INVALID_PUBLIC_KEY = "integration.authorization.invalid.publickey";
-
-  private static final String INVALID_PUBLIC_KEY_SOLUTION = INVALID_PUBLIC_KEY + ".solution";
-
-  @Autowired
-  private LogMessageSource logMessage;
-
-  /**
-   * Creates a RSA signer based on a given private key.
-   * @param privateKey private key in PKCS8 syntax.
-   * @return OAuthRsaSigner
-   */
-  public OAuthRsaSigner getOAuthRsaSigner(String privateKey) throws OAuth1Exception {
```

Comment:
You might keep this class just to deal with OAuth stuff. The other methods should be moved to RsaKeyUtils.